build/bazel/remote/execution/v0.2/remote_execution.proto

```diff
@@ -760,15 +767,16 @@ message ExecuteRequest {
   // omitted.
   string instance_name = 1;
 
-  // The action to be performed.
-  Action action = 2;
-
   // If true, the action will be executed anew even if its result was already
   // present in the cache. If false, the result may be served from the
   // [ActionCache][build.bazel.remote.execution.v0_2.ActionCache].
   bool skip_cache_lookup = 3;
 
-  reserved 4, 5; // Used for removed fields in an earlier version of the API.
+  reserved 2, 4, 5; // Used for removed fields in an earlier version of the API.
+
+  // The digest of the [Action][build.bazel.remote.execution.v0_2.Action] to
+  // execute.
+  Digest action_digest = 6;
```

Comment:
We also need to update the comment on UpdateActionResult to say that the server may require the action and its part to be uploaded into the CAS.

Reply:
I did, it's above.